.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,40 @@
-## What changes were proposed in this pull request?
+<!--
+Thanks for sending a pull request!  Here are some tips for you:
+  1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+  2. Ensure you have added or run the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+  3. If the PR is unfinished, add '[WIP]' in your PR title, e.g., '[WIP][SPARK-XXXX] Your PR title ...'.
+  4. Update your PR description whenever it is needed.
```

Comment:
At first glance this might be a little unclear. I think what you are getting at is that if anything changes in the course of the PR, keep the description updated?
If so then maybe change to "Be sure to keep the PR description updated to reflect all changes"?